src/pudl/package_data/eia860/column_maps/boiler_info.csv

```diff
@@ -0,0 +1,39 @@
+year_index,2009,2010,2011,2012,2013,2014,2015,2016,2017,2018,2019,2020
+utility_id_eia,utility_id,utility_id,utility_id,utility_id,utility_id,utility_id,utility_id,utility_id,utility_id,utility_id,utility_id,utility_id
+utility_name_eia,utility_name,utility_name,utility_name,utility_name,utility_name,utility_name,utility_name,utility_name,utility_name,utility_name,utility_name,utility_name
+plant_id_eia,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code
+plant_name_eia,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name
+state,state,state,state,state,state,state,state,state,state,state,state,state
+boiler_id,boiler_id,boiler_id,boiler_id,boiler_id,boiler_id,boiler_id,boiler_id,boiler_id,boiler_id,boiler_id,boiler_id,boiler_id
+hrsg,name_of_water_source,name_of_water_source,,,,,,hrsg,hrsg,hrsg,hrsg,hrsg
+boiler_status,boiler_status,boiler_status,boiler_status,boiler_status,boiler_status,boiler_status,boiler_status,boiler_status,boiler_status,boiler_status,boiler_status,boiler_status
+type_of_boiler,type_of_boiler,type_of_boiler,type_of_boiler,type_of_boiler,type_of_boiler,type_of_boiler,type_of_boiler,type_of_boiler,type_of_boiler,type_of_boiler,type_of_boiler,type_of_boiler
```

Comment:
Should be `boiler_type` to match naming of other columns with `_type` as a suffix (if it's a descriptive string).